driver.go

```diff
@@ -379,42 +410,62 @@ func (d *Driver) GetState() (state.State, error) {
 
 // Kill stops a host forcefully
 func (d *Driver) Kill() error {
-	// TODO find something more forceful than "Stop"
 	return d.Stop()
 }
 
 // Remove a host
 func (d *Driver) Remove() error {
-	client, err := d.client()
+	c, err := d.client()
 	if err != nil {
 		return err
 	}
-	return client.DeleteMachine(d.TritonMachineId)
+
+	ctx := context.Background()
+	input := &compute.DeleteInstanceInput{
+		ID: d.TritonMachineId,
```

Comment:
This is all backwards compatible, right?  If someone had an older "state" which included a `TritonMachineId` it'll still work with this new driver, right?

Reply:
Good question, I would strongly assume so given that the a Triton machine ID is a UUID which is global in Triton itself.